src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -118,32 +118,33 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CannotFindAnyManifestsFileSearched" xml:space="preserve">
-    <value>Cannot find any manifests file. Searched {0}</value>
+    <value>Cannot find any manifests file. Searched
```

Comment:
Just want to confirm the line break here is intentional. If that's the case, should we have some : separator or something like that.

Reply:
: it good

Reply:
yes, the line break is intentional and there will be several indented new lines for detail. I added `:` at the end